massgov/mayflower
# DP-9570 Remove raw filter from _patterns/01-atoms/08-lists/unordered-list.twig

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Remove raw filter from _patterns/01-atoms/08-lists/unordered-list.twig.

Usage of the template in Mayflower
```
 ~/MassIT/mayflower/ [feature/DP-9570_atoms-lists-unordered-list] grep -r 'unordered-list.twig' patternlab/styleguide/source
patternlab/styleguide/source/_patterns/03-organisms/by-author/form-requirements-in-two-column.json:        "path": "@atoms/08-lists/unordered-list.twig",
patternlab/styleguide/source/_patterns/03-organisms/by-author/form-requirements.json:        "path": "@atoms/08-lists/unordered-list.twig",
patternlab/styleguide/source/_patterns/03-organisms/by-author/rich-text-description.json:      "path": "@atoms/08-lists/unordered-list.twig",
patternlab/styleguide/source/_patterns/03-organisms/by-author/rich-text.json:      "path": "@atoms/08-lists/unordered-list.twig",
patternlab/styleguide/source/_patterns/03-organisms/by-author/steps-unordered.json:          "path": "@atoms/08-lists/unordered-list.twig",
patternlab/styleguide/source/_patterns/04-templates/01-content-types/alert.json:          "path": "@atoms/08-lists/unordered-list.twig",
patternlab/styleguide/source/_patterns/04-templates/01-content-types/form-page.json:        "path": "@atoms/08-lists/unordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/event~end-date-with-time.json:        "path": "@atoms/08-lists/unordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/event~end-date.json:        "path": "@atoms/08-lists/unordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/guide.json:            "path": "@atoms/08-lists/unordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/guide.json:                    "path": "@atoms/08-lists/unordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/howto.json:        "path": "@atoms/08-lists/unordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/howto.json:        "path": "@atoms/08-lists/unordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/policy-advisory-directive.json:        "path": "@atoms/08-lists/unordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/site-alert.json:          "path": "@atoms/08-lists/unordered-list.twig",
 ~/MassIT/mayflower/ [feature/DP-9570_atoms-lists-unordered-list]
```

## Related Issue / Ticket

- [DP-9570](https://jira.mass.gov/browse/DP-9570)


## Steps to Test


1. Go to the following page type samples and see the areas the template is used have any display issues in both Mayflower and Drupal.

- Event:  You will need
- Guide:  section unit
- How-to:  main content area
- Advisory:  section unit
- Site-alert: Alert unit content

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
